src/guides/v2.2/cloud/live/live.md

```diff
@@ -37,33 +33,51 @@ We strongly recommend testing in these environments due to the complexity of you
 
 ## Set up Magento Security Scan Tool {#security-scan}
 
-The Magento Security Scan Tool enables you to regularly monitor your store websites and receive updates for known security risks, malware, and out of date software. This is a free service available for all implementations and versions of {{site.data.var.ece}}. You access the tool through your [Magento Marketplace account](https://account.magento.com/customer/account/login).
+The Magento Security Scan Tool enables you to regularly monitor your store websites and receive updates for known security risks, malware, and out-of-date software. This is a free service available for all implementations and versions of {{site.data.var.ece}}. You access the tool through your [Magento Marketplace account](https://account.magento.com/customer/account/login).
 
 *  Monitor your sites security status and applied security updates
-*  Receive security updates and site specific notifications
+*  Receive security updates and site-specific notifications
 
-For detailed instructions to set up and perform scans, see the [Magento User Guide](http://docs.magento.com/m2/ee/user_guide/magento/security-scan.html). Typically, you want to start using this tool as you enter UAT testing.
+For detailed instructions to set up and perform scans, see the [Magento User Guide](http://docs.magento.com/m2/ee/user_guide/magento/security-scan.html). Typically, you want to start using this tool as you enter user acceptance testing (UAT).
 
-Each site to be scanned must be registered through Magento Security Scan tab. This registration process includes acceptance of Magento’s disclaimer prior to scanning. You control both scan scheduling and the authorization of personnel to be notified when each scan is completed. Scans can be scheduled for either a specific, recurring date and time or on-demand as required.
+Each site you scan must be registered through the Magento Security Scan tab. The registration process requires you to accept Magento’s disclaimer prior to scanning. You control both scan scheduling and the authorization of personnel to be notified when each scan is complete. You can schedule scans for a specific, recurring date and time or on demand as required.
 
+{%
+include note.html
+type='info'
+content='The security scan tool uses the following public IP addresses:
+
+```text
+52.72.230.169
+52.86.204.1
+52.87.98.44
+```
+
+You must whitelist these IP addresses in your network firewall rules to allow the tool to scan your site.'
+%}
```

Comment:
Wonder if it would be better to put the note higher up, or list it as a prerequisite step instead of a note, so that customers don't have to backtrack after going through registration process. Also, the person registering might not be the same person to whitelist the addresses, so putting at a prerequisite gives time to get the task completed.

Reply:
done

Reply:
1) agree with @meker12 . 
2) Add: Magento security scan tool posts requests to ports 80 and 443 only.
3) Also please add the following section: Magento security scanner uses several user agent strings to simulate real-life malware activity. You might see those user agents in your analytics or access logs:

Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:57.0) Gecko/20100101 Firefox/57.0
GuzzleHttp/6.3.3 curl/7.29.0 PHP/7.1.18
Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/37.0.2062.120 Safari/537.36
Visbot/2.0 (+http://www.visvo.com/en/webmasters.jsp;bot@visvo.com)

Reply:
done